src/Boilerplates.php

```diff
@@ -0,0 +1,99 @@
+<?php
+
+namespace JustCoded\WP\Composer;
+
+use Composer\IO\IOInterface;
+use Composer\Script\Event;
+
+class Boilerplates {
+
+	/**
+	 *  Arguments array
+	 * @var $args
+	 */
+	public static $args = array();
+
+
+	/**
+	 * @param Event $event
+	 *
+	 * @return bool
+	 */
+	public static function theme( Event $event ) {
+		$io = $event->getIO();
+
+		$theme_name = str_replace( 't=', '', $event->getArguments()[1] );
+		$theme_slug = str_replace( ' ', '', $theme_name );
+		$theme_dir  = str_replace( ' ', '', $event->getArguments()[0] );
+		if ( empty( $theme_dir ) ) {
+			$theme_dir = 'wp-content/themes';
+		}
+
+		$src  = 'vendor/justcoded/wordpress-theme-boilerplate';
+		$dst  = $theme_dir . '/' . $theme_slug;
+		self::$args = array(
+			'full_path_to_new_theme' => $dst,
+			'theme_slug'             => $theme_slug,
+			'theme_name'             => $theme_name,
+		);
```

Comment:
using static vars is not good way to pass variables through code. this is the same as using global. this is wrong

Reply:
done